ts/packages/ts/src/stores/KeyValue.ts

```diff
@@ -14,10 +14,7 @@ export abstract class KeyValueConfigStore extends ConfigStore {
   protected abstract delete(key: string): Promise<void>;
 
   private calcKey({ set, key }: ConfigStoreQuery): string {
-    if (!set) {
-      return key;
-    }
-    return set;
+    return set || key;
```

Comment:
I know it doesnt matter but i like the strict if syntax more

Reply:
sure. reverted